From 3b544789da07d84b61fdeb5c8801f6b5accfe03a Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:27:02 -0400
Subject: [agp] ati: Use PCI_COMMAND instead of hard-coded 4

Message-id: <20140719042701.19438.49301.stgit@gir.stowe>
Patchwork-id: 86003
O-Subject: [RHEL7.1 PATCH 41/81] agp/ati: Use PCI_COMMAND instead of hard-coded 4
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 167b1f049008b367a9003a6a8df090af4282a6b0

commit 167b1f049008b367a9003a6a8df090af4282a6b0
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Mon Jan 6 16:15:31 2014 -0700

    agp/ati: Use PCI_COMMAND instead of hard-coded 4

    We're accessing the PCI_COMMAND register here, so use the appropriate
    #define.  The bit we're writing (1 << 14) isn't defined by the PCI or PCIe
    spec, so we don't have a name for it.

    No functional change.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/char/agp/ati-agp.c b/drivers/char/agp/ati-agp.c
index 2157643..7f1fd28 100644
--- a/drivers/char/agp/ati-agp.c
+++ b/drivers/char/agp/ati-agp.c
@@ -221,8 +221,8 @@ static int ati_configure(void)
  readl(ati_generic_private.registers+ATI_GART_FEATURE_ID); /* PCI Posting.*/
 
  /* SIGNALED_SYSTEM_ERROR @ NB_STATUS */
- pci_read_config_dword(agp_bridge->dev, 4, &temp);
- pci_write_config_dword(agp_bridge->dev, 4, temp | (1<<14));
+ pci_read_config_dword(agp_bridge->dev, PCI_COMMAND, &temp);
+ pci_write_config_dword(agp_bridge->dev, PCI_COMMAND, temp | (1<<14));
 
  /* Write out the address of the gatt table */
  writel(agp_bridge->gatt_bus_addr, ati_generic_private.registers+ATI_GART_BASE);
-- 
1.7.1